src/data.js

```diff
@@ -74,3 +74,13 @@ module.exports.Build = Entity.configure({
     return item;
   },
 });
+
+module.exports.OwnersDirectory = Entity.configure({
+  version: 1,
+  partitionKey: Entity.keys.StringKey('owner'),
+  rowKey: Entity.keys.ConstantKey('someConstant'),
+  properties: {
+    installationID: Entity.types.Number,
```

Comment:
Typically we would spell this `installationId` (I know, really minor detail...)

If this is easy to change now, let's fix it.  If it reqiures a new `.configure({version: 2, ..})` then it's not worth ith.